### PR TITLE
Add Randomly Assign Task description to DG

### PR DESCRIPTION
**Pull request by mjgui**
Help to check whether UML is valid, thanks!

### CODE REVIEW

File: docs/diagrams/RandomlyAssignTaskActivityDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+if () then ([else])
+    :Retrieve task at given index;
+    if () then ([else])
+        :Insert all unassigned team members to a list;
+        :Assign task to one random member from list;
+    else ([all team members already assigned task])
+        :Throw CommandException;
+    endif
+else ([task index is out of bounds])
+    :Throw CommandException;
```
Comment (Devanshshah1309) [inline docs/diagrams/RandomlyAssignTaskActivityDiagram.puml:15]: Looks mostly okay, just a few observations:

1. Would be better to have arrows (navigability) for the other UML diagrams
2. Change `Displays` to `Display` in activity diagram.
3. Throw `ParseException` when `taskIndex` is invalid (not out of bounds, but simply invalid e.g. string) -> can refer to `AddTaskActivityDiagram`
4. Can phrase the "Insert all unassigned team members to list" better to something like "Randomly choose a member from team members that have not been assigned to this task" and then remove the next step.

File: docs/diagrams/RandomlyAssignTaskActivityDiagram.puml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+title Activity Diagram for Randomly Assign Task
+
+if () then ([else])
+    if () then ([else])
+        :Retrieve task at given index;
+        if () then ([else])
+            :Assign task to randomly chosen
+            team member that has not been
+            assigned this task yet;
+        else ([all team members already assigned task])
```
Comment (Devanshshah1309) [inline docs/diagrams/RandomlyAssignTaskActivityDiagram.puml:13]:
```suggestion
        else ([all team members are already assigned to this task])
```

File: docs/diagrams/RandomlyAssignTaskState1.puml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+object "__person2:Person__" as Person2 <<Person>>
+object "__person3:Person__" as Person3 <<Person>>
+object "__:Task__" as Task <<TaskType>>
+
+Team --> Person1
+Team --> Person2
+Team --> Person3
+Team --> Task
+Task --> Person2
+
```
Comment (Devanshshah1309) [inline docs/diagrams/RandomlyAssignTaskState1.puml:27]: The png doesn't appear to be updated btw since arrows are still not shown

Comment (mjgui) [reply docs/diagrams/RandomlyAssignTaskState1.puml:27]: good catch

**Review by Devanshshah1309 — APPROVED** [review]

**Pull request merged**